doc/liquid_conditions.md

```diff
@@ -0,0 +1,67 @@
+# Liquid conditions examples
+
+When using liquid, a lot of things can be achieved, and here are some examples
+of conditions that can be used across multiple policies
+
+
+## Headers
+
+### Header is not present
+
+Condition pass if header 'Backend' is sent and it's staging
```

Comment:
```suggestion
Condition passes if the  'Backend' header is sent and it's staging
```

Questions:
a. Who/What sends the  'Backend' header ?
b. What does `staging` refer to?